t/14-grutasks-git.t

```diff
@@ -95,8 +114,11 @@ subtest 'git clone' => sub {
 
     subtest 'no default remote branch' => sub {
         $ENV{OPENQA_GIT_CLONE_RETRIES} = 0;
-        $clone_dirs = {'/tmp/' => 'http://localhost/nodefault.git'};
-        my $res = run_gru_job($t->app, 'git_clone', $clone_dirs, {priority => 10});
+        $clone_dirs = {"$git_clones/nodefault" => 'http://localhost/nodefault.git'};
+        stderr_like {
+            $res = run_gru_job($t->app, 'git_clone', $clone_dirs, {priority => 10})
+        }
+        qr(Error detecting remote default), 'error on stderr';
```

Comment:
As a similar block repeats very often I suggest to shorten it so that the repeated line is only a single line. How about


```suggestion
@args = ($t->app, 'git_clone', $clone_dirs, {priority => 10});
…
        stderr_like { $res = run_gru_job(@args) } qr(Error detecting remote default), 'error on stderr';
```

if that's valid syntax :)

Reply:
Ok. I had to make some adjustments for the changing `$clone_dirs` values.